docs/Extending_Input_Rendering_bcee26a.md

```diff
@@ -19,18 +19,18 @@ The control API and the `render` method can be inherited as it is and the `rende
 ``` js
   
 sap.m.Input.extend("HighlightInput", {// call the new Control type "HighlightInput" 
-								// and let it inherit from sap.m.Input
+                                      // and let it inherit from sap.m.Input
 
 			renderer: {
 				// note that no render() function is given here. The Input's render() function is used. 
 				// But one function is overwritten:
 
 				writeInnerAttributes : function(oRm, oInput) {
 					sap.m.InputRenderer.writeInnerAttributes.apply(this, arguments); // the default method should be called
-																		 // this will make sure that all default input attributes will be there
+					                                                                 // this will make sure that all default input attributes will be there
 
 					oRm.addStyle('background-color', '#ffff00');  // this change could also be done with plain CSS. 
-														  // But you get the idea...
+					                                              // But you get the idea...
 					  }
```

Comment:
Thank you, Jan! You are right that it should be aligned.
Can I suggest to keep the tabs and move them to one line lower, so that they fit most screens without a horizontal scrollbar, like this:
```suggestion
sap.m.Input.extend("HighlightInput",{
// call the new Control type "HighlightInput" 
// and let it inherit from sap.m.Input

			renderer: {
			// note that no render() function is given here. The Input's render() function is used. 
			// But one function is overwritten:

				writeInnerAttributes : function(oRm, oInput) {
					sap.m.InputRenderer.writeInnerAttributes.apply(this, arguments);
					// the default method should be called
					// this will make sure that all default input attributes will be there

					oRm.addStyle('background-color', '#ffff00');
					// this change could also be done with plain CSS. 
					// But you get the idea...
					  }
				  }
			  });
```

Reply:
Your suggestion does not seem correct? It adds a bunch of code?

And I don’t quite get what you are suggesting from your comment.

Reply:
Sorry, I should've added a comment and not a suggestion. This is what I meant:
``` js
sap.m.Input.extend("HighlightInput",{
// call the new Control type "HighlightInput" 
// and let it inherit from sap.m.Input

			renderer: {
			// note that no render() function is given here. The Input's render() function is used. 
			// But one function is overwritten:

				writeInnerAttributes : function(oRm, oInput) {
					sap.m.InputRenderer.writeInnerAttributes.apply(this, arguments);
					// the default method should be called
					// this will make sure that all default input attributes will be there

					oRm.addStyle('background-color', '#ffff00');
					// this change could also be done with plain CSS. 
					// But you get the idea...
					  }
				  }
			  });
```

Reply:
You suggest to keep tabs but my suggested change does not remove the indentation tabs. So I don’t get what you mean by that.

From the code in your comment I take you suggest to move the comments into separate lines?

Is comments below code convention in this repository? I would expect them to be before the code it comments.

This change does not in fact increase the line width. Are you suggesting to fix this unrelated issue alongside the indentation issue?

I can’t see guidelines regarding line length in CONTRIBUTING.md but personally I certainly wouldn’t mind putting the comments in front. (I found this style of commenting pretty confusing/suboptimal in the first place.) But as I said I would expect them upfront.